### PR TITLE
Add a Japanese tutorial with usage of `<ruby>`

### PR DESCRIPTION
**Pull request by knagasaki**

### CODE REVIEW

File: tutorial_ja/README.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+このチュートリアルでは、CETEIceanを用いてTEIファイルを公開するための一連のステップを解説します。TEI P5の形式の（大きいですが）単純なファイル、`fpn-washington.xml`を用いて解説を始めます。我々はこのファイルをWebブラウザに表示することを目指します。
+
+
+First, a note about viewing the results of your work: CETEIcean's default method for displaying TEI relies on loading a TEI file from another location. Not all browsers will allow you to do this when you view an HTML file directly on your file system. You should try it, but if it doesn't work, then you will have to run a web server, put your files on a web server, or use a text editor with preview capabilities. [Atom](https://atom.io), with the `atom-html-preview` plugin is the example we will use for this tutorial, but there are many other options. You should download and install Atom, or an equivalent text editor, before starting this tutorial. A text editor is different from other programs you may already use for editing 'text', such as LibreOffice or Word, in that it edits only plain text files.
```
Comment (raffazizzi) [inline tutorial_ja/README.md]: Is the English paragraph left here intentionally or should it be removed?

File: tutorial_ja/README.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+上記のCSSの最初のセレクタ（tei-list[type=simple]の部分）は `<tei-list type="simple">` 要素にマッチし、リストアイテムが箇条書きや数字で装飾されてはならないことを「list-style-type: none」で指定しています。2 番目のセレクタ（tei-list[type=simple]>tei-itemの部分）は `<tei-list>` 要素をリストアイテムとして識別します (HTML の `<li>` と同じです)。この変更を行ってからプレビューを再読み込みすると、目次がリストとしてフォーマットされているのがわかると思います。余白や別のスタイルを追加して、見栄えをよくするために実験することもできます。また、`<hi rend="italics">` の要素も同様にして簡単に扱うことができます：
+```css
+tei-hi[rend=italics] {
+  font-style: italic;
+}
+```
+
+改行（Line beginning）はちょっとややこしく、CSSの「擬似セレクタ」が必要です。
+Line beginnings are a little trickier, and require a CSS "pseudo-selector":
```
Comment (raffazizzi) [inline tutorial_ja/README.md]: Same here, was this left intentionally?

**Comment by raffazizzi** [comment]
Thank you so much for translating the tutorial! Before merging, I just wanted to confirm whether some of the English text in the tutorial was left there intentionally.

File: tutorial_ja/README.md (hunk 2 above)
Comment (knagasaki) [reply tutorial_ja/README.md]: I'm sorry for my mistake. The English sentences have been deleted.

**Comment by raffazizzi** [comment]
Thank you! Your contribution also prompted me to review the English tutorial and I made two small updates:
* using VSCode instead of Atom (Atom is being discontinued soon)
* using python3 or node instead of python 2 to run a server

You can see a summary of changes here: https://github.com/TEIC/CETEIcean/pull/58/files

Would you be able to update the JP tutorial accordingly?

**Comment by knagasaki** [comment]
Thank you for the notification. I revised it, following the new version.

**Comment by raffazizzi** [comment]
Perfect, thank you! Merged.

**Pull request merged**